src/mmw/js/src/modeling/views.js

```diff
@@ -91,11 +110,7 @@ var ScenariosView = Marionette.LayoutView.extend({
     },
 
     addScenario: function() {
-        var scenario = new models.ScenarioModel({
-                name: 'New Scenario'
-            });
-        this.collection.add(scenario);
-        this.collection.setActiveScenario(scenario.cid);
+        this.triggerMethod('create:scenario');
```

Comment:
Same comment as above, we don't need to add a layer of indirection if the new helper methods are on the collection instead of the project.